functions/go/set-image/image_transformer_test.go

```diff
@@ -30,15 +30,16 @@ func runImageTransformerResults(input, config string) (*fn.ResourceList, error)
 	}
 	rl.FunctionConfig = functionConfig
 	in, _ := rl.ToYAML()
-	out, err := fn.Run(fn.ResourceListProcessorFunc(setImageTags), in)
+	out, err := fn.Run(fn.ResourceListProcessorFunc(SetImageTagSDK), in)
```

Comment:
Using "SDK" can be confusing. The `SetImageTagSDK` supposes to transform the KubeObject-based resources with "image" logic. It is not and we have no plan to use it as a SDK.